webui/react/src/pages/TrialLogs.tsx

```diff
@@ -149,19 +149,18 @@ const TrialLogs: React.FC = () => {
     return <Message title={`Unable to find Trial ${trialId}`} type={MessageType.Warning} />;
   }
 
-  const experimentDetailPath = `/experiments/${trial.data?.experimentId}`;
+  const experimentDetailPath = `/experiments/${experimentId}`;
   const trialDetailPath = `${experimentDetailPath}/trials/${trialId}`;
```

Comment:
**suggestion:** under `routes/utils.ts` there is an object `paths` with functions to generate paths. Could you add these to that and use that instead? There is one for `trialDetails` already so far, but could use an `experimentDetails` one too.